resources/docker-images/linux/run-tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -e
+
```

Comment:
We could add a test to stop this if not running in a Docker image, for user safety